aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -338,25 +356,34 @@ module aptos_framework::account {
 
         // Verify the given `from_public_key_bytes` matches this account's current authentication key.
         if (from_scheme == ED25519_SCHEME) {
-            let from_pk = ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
-            let from_auth_key = ed25519::unvalidated_public_key_to_authentication_key(&from_pk);
+            let from_pk =
+                ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
+            let from_auth_key = ed25519::unvalidated_public_key_to_authentication_key(
+                &from_pk
+            );
```

Comment:
This is actually interesting, the first line chooses one approach, the second chooses a different approach.

I prefer approach 2
```suggestion
            let from_pk = ed25519::new_unvalidated_public_key_from_bytes(
                from_public_key_bytes
            );
            let from_auth_key = ed25519::unvalidated_public_key_to_authentication_key(
                &from_pk
            );
```

Reply:
This should have precisely triggered two rules within the movefmt code, resulting in different styles. The first style is the rustfmt style, and the second is the general style. We could consider completely removing the first style; in the last discussion, it seemed that many people did not particularly like the first style.